README.md

```diff
@@ -1,9 +1,26 @@
-# strfmt: rust library for formatting dynamic strings
+# Index  
```

Comment:
Put the index after the first line + intro and make it `##`

Also rename it "Table of Contents". Otherwise I like it :D

Reply:
Done